app/models/dunning_campaign.rb

```diff
@@ -30,6 +30,25 @@ class DunningCampaign < ApplicationRecord
   def self.ransackable_attributes(_auth_object = nil)
     %w[name code]
   end
+
+  def reset_customers_last_attempt
```

Comment:
Should this be part of the service rather than of a model method?

Reply:
Well.. - we plan to use this method in different places, that's why I decided to put it on the model..